### ISSUE TITLE
UI?

### ISSUE THREAD

**Issue by munir131** [munir131/attachment-downloader#3]
Planning to use UI based application for end user. So, no need to code and enhance UX. Currently, I plan to use Electron as we can get two benefits
1) Cross platform
2) Portable

**Comment by attiq-khan** [comment]
Can we search by email id of sender ?

**Comment by attiq-khan** [comment]
And UI will be great help. :1st_place_medal:

**Comment by munir131** [comment]
@attiq-khan Yes. You can search - https://github.com/munir131/attachment-downloader/blob/b96718a3f86b2067cb936a02df283f4ea7cf7880/index.js#L230
Yes. UI can help but I am not getting enough time to implement it

**Comment by attiq-khan** [comment]
so index.js --email Id abc@gmail.com will download all attachments ?

**Comment by munir131** [comment]
@attiq-khan No. Just run `node index.js`, it will ask you for further option where you can mention email

**Comment by attiq-khan** [comment]
Sure Thanks for quick reply :) I will check and let you know.
Thanks again for sharing this :)
